quick_blue_windows/windows/quick_blue_windows_plugin.cpp

```diff
@@ -72,6 +72,24 @@ std::string to_uuidstr(winrt::guid guid) {
   return std::string{ chars };
 }
 
+static IAsyncOperation<GattDeviceServicesResult> GetGattServicesAsync(BluetoothLEDevice device) {
```

Comment:
I created these as free functions because they have limited association with the class. Not sure about the naming convention, but this was convenient.